artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionBridge.java

```diff
@@ -351,10 +355,15 @@ private String appendIgnoresToFilter(String filterString) {
       }
       filterString += "!" + storeAndForwardPrefix;
       filterString += ",!" + managementAddress;
-      filterString += ",!" + managementNotificationAddress;
       return filterString;
    }
 
+   private String createPermissiveManagementNotificationToFilter() {
+      StringBuilder filterBuilder = new StringBuilder(ManagementHelper.HDR_NOTIFICATION_TYPE).append(" = '")
```

Comment:
Whats this trying to achieve?

Reply:
The inter-cluster notification message needs to specify the routing address to route the message. Different from the notifications of CONSUMER_CREATE and BINDING_ADD, the routing address (the address created or consumed by itself) has been determined before the notification is sent. When sending a SESSION_CREATED notification, We can't know which address to use, so I chose to use the `ManagementManagementAddress`, which is consumed by each node in the cluster-connection.
But the notification message using the `ManagementNotificationAddress` address is filtered by the filter, so I modified the filter and allow SESSION_CREATED notification to be routed to other nodes in the cluster.

Reply:
Although it is possible to add a dedicated address to the SESSION_CREATED notification when the cluster-connection is created, it is not necessary to do so.